docs/consume-packages/install-use-packages-powershell.md

```diff
@@ -1,178 +1,205 @@
 ---
-title: Install and manage NuGet packages using the console in Visual Studio
-description: Instructions for using the NuGet Package Manager Console in Visual Studio for working with packages.
+title: Manage NuGet packages with the Visual Studio Package Manager Console
+description: See how to work with NuGet packages by using PowerShell commands in the Visual Studio Package Manager Console.
 author: JonDouglas
 ms.author: jodou
-ms.date: 07/08/2019
+ms.date: 08/17/2020
 ms.topic: conceptual
 f1_keywords: 
   - "vs.nuget.packagemanager.console"
 ---
 
-# Install and manage packages with the Package Manager Console in Visual Studio (PowerShell)
+# Manage packages with the Visual Studio Package Manager Console (PowerShell)
 
-The NuGet Package Manager Console lets you use [NuGet PowerShell commands](../reference/powershell-reference.md) to find, install, uninstall, and update NuGet packages. Using the console is necessary in cases where the Package Manager UI does not provide a way to perform an operation. To use `nuget.exe` CLI commands in the console, see [Using the nuget.exe CLI in the console](#use-the-nugetexe-cli-in-the-console).
+The Package Manager Console in Visual Studio uses PowerShell commands to interact with NuGet packages. You can use the console when there's no way to do an operation through the [Package Manager UI](install-use-packages-visual-studio.md). You can also use [dotnet CLI](../reference/dotnet-commands.md) or [NuGet CLI](#use-the-nugetexe-cli-in-the-console) commands in the console.
```

Comment:
I think it's still be great to link the powershell reference.

Reply:
@nkolev92 - It's linked in the next paragraph.